### ISSUE TITLE
Layout First Version

### ISSUE THREAD

**Issue by fcbaila** [ipfs/conf#28]
We are already working on what could be the final site layout, following the branding colors, and thought about flipping the colors between pages to create contrast between them.

Homepage:
<img width="1440" alt="homepage 2x" src="https://user-images.githubusercontent.com/6978461/40192652-e92a56b4-59fc-11e8-8af9-45cdfabb5807.png">

IPFS Developers Meeting:
<img width="1440" alt="ipfs meeting - copy 2x" src="https://user-images.githubusercontent.com/6978461/40192660-efd47bb6-59fc-11e8-915d-1049d553af15.png">

libp2p Developers Meeting:
<img width="1440" alt="libp2p meeting 2x" src="https://user-images.githubusercontent.com/6978461/40192668-f65b2656-59fc-11e8-85ee-903796bb8510.png">

**Comment by mgoelzer** [comment]
Let's avoid anything that visually puts the Developers Meeting at the same level of importance as IPFS Conf (even though they are chronologically next).  

The website is about IPFS Conf in November.  The majority of people who visit this website should come to IPFS Conf and will never even care that the Developers Meetings took place.

<img width="983" alt="layout_first_version_ _issue__28_ _ipfs_conference" src="https://user-images.githubusercontent.com/225111/40123206-add09b1e-58f3-11e8-88a9-a4559dafac4e.png">

**Comment by fcbaila** [comment]
@mgoelzer New version: 

<img width="1440" alt="homepage 2x" src="https://user-images.githubusercontent.com/6978461/40126678-bd4d3a2a-5925-11e8-8679-358ebc49f11b.png">

**Comment by inesssilva** [comment]
libp2p page

<img width="1440" alt="libp2p meeting 2x" src="https://user-images.githubusercontent.com/17513109/40130985-7a5652a0-5930-11e8-98e6-ff4147224201.png">

**Comment by fcbaila** [comment]
Homepage new version:

<img width="1440" alt="homepage 2x" src="https://user-images.githubusercontent.com/6978461/40189663-ea2e77fa-59f4-11e8-8757-3f7b1a031043.png">

**Comment by fcbaila** [comment]
We have moved forward with the layouts and tried to invert the colors for each page, creating a distinction between them and the events themselves. 

<img width="1440" alt="homepage 2x" src="https://user-images.githubusercontent.com/6978461/40190031-b658298e-59f5-11e8-9c52-a6d789b15454.png">
<img width="1440" alt="ipfs meeting - copy 2x" src="https://user-images.githubusercontent.com/6978461/40190032-b6777e6a-59f5-11e8-9331-15114fbcf375.png">
<img width="1440" alt="libp2p meeting 2x" src="https://user-images.githubusercontent.com/6978461/40190033-b6936a12-59f5-11e8-9c30-3d797c25aac5.png">

**Comment by meiqimichelle** [comment]
Content and information architecture feedback for this layout is now tracked separately in #34 . I will make a separate issue to provide feedback on mood/branding/visual look and feel.

**Comment by meiqimichelle** [comment]
Check that, maybe it makes the most sense to provide visual look and feel feedback here in this issue instead of in a separate one. Here we go!

1) We didn't get you the latest visual design guidelines when you created this first version, apologies. The latest direction (which is pretty lightweight) is in this [ipfs-css repo](https://github.com/ipfs-shipyard/ipfs-css/blob/master/theme.json). For a quick grab of colors and typefaces, go [here](https://github.com/ipfs-shipyard/ipfs-css/blob/master/theme.json). Montserrat is the heading typeface, and Inter-UI is the general-purpose body copy typeface.

2) @akrych is going to do another iteration on the conference logo in #33, but she is out for the rest of the week. You can read that issue to see the direction we asked her to explore (the speech-bubble option, not the various thin line-and-dot options). Since she's out, however, please just leave a placeholder for a future mark in your designs, but don't let that hold up your next iteration. Please also explore designs that focus less on the line-dot, sparse feel, and more towards richer designs that inspire ideas of creating, building, community, shared enthusiasm, sharing information, working together.

To be clear, this is the mark that @akrych will iterate on:
<img width="69" alt="screenshot 2018-05-23 15 41 53" src="https://user-images.githubusercontent.com/4827522/40431688-d74e0a7e-5e9f-11e8-8b4b-299d5d8b0366.png">

3) Our family of websites all have a bold, center-aligned splash as the first thing you see on their homepages. Please explore designs that maintain, support, or echo this approach. (See https://ipfs.io/ and https://libp2p.io/ and https://protocol.ai/ for examples.) However, while we would like to make this site seem part of the system, we are still definitely looking to you to suggest ideas for a striking first look that will help get folks excited about the conference. Please feel free to go bold for that homepage! The 'Community/Dev meeting' sub-page can be a straightforward, more content-delivery sort of page, if you like, by comparison.

Please let me know if any of this isn't clear, or if you'd like any other feedback or direction on specific items in your v1.

**Comment by meiqimichelle** [comment]
Hi @inesssilva , nice to meet you here!

The information architecture and content has changed significantly, as per #34, since you designed this first iteration of the layout. We can't retrofit the content to this design, and of course design should always follow content to create a clear user experience.

Still, we understand the timeline is short, and we were slow in providing feedback. **Would you be able to iterate on the design if we extend the launch date by one week?**

**Comment by inesssilva** [comment]
Yes, I would suggest to extend the deadline to 1 Jun

To sum up, please confirm these changes:

1. Information architecture & content 
2. Colours and Typefaces
3. Bold, center-aligned splash homepage

**Comment by meiqimichelle** [comment]
Ah! @inesssilva I believe there was a miscommunication on the original deadline. We have a temporary blog post up now that is serving to get the word out on the upcoming meetings and conferences. We **do not** need to launch this week, I'm not sure where that deadline came from. Let's target June 1, as you suggest.

To confirm: the next step is another design iteration that reflects the information architecture and content in #34 and the three points in [this comment in this issue](https://github.com/ipfs/conference/issues/28#issuecomment-391376745).

One note: please do not start development until we've come to agreement on the design direction, unless you prefer to design in code while still allowing iterations. Thank you!

**Comment by inesssilva** [comment]
Hi @meiqimichelle, I don't know what happened, but perfect. 

I have a new design iteration for homepage. I'm uploading 3 versions because I wasn't sure if the info about "next events" it's still relevant since it's not on content docs anymore. 

Also, I think the center-aligned splash doesn't fit properly, but I will leave it here for you to see which version you prefer.

Version 1
<img width="1440" alt="homepage v2 - 1 2x" src="https://user-images.githubusercontent.com/17513109/40500846-029538ea-5f7e-11e8-8295-5e0ed44826ca.png">

Version 2
<img width="1440" alt="homepage v2 - 2 2x" src="https://user-images.githubusercontent.com/17513109/40500849-02c488a2-5f7e-11e8-8c28-4b86a31045b1.png">

Version 3
<img width="1440" alt="homepage v2 - 3 2x" src="https://user-images.githubusercontent.com/17513109/40500850-02f52106-5f7e-11e8-96da-e44f0c142d97.png">
